src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.Toolkit.Uwp.UI.Lottie
 {
-	public abstract partial class LottieVisualSourceBase : DependencyObject, IAnimatedVisualSource
+	public abstract partial class LottieVisualSourceBase : DependencyObject, IAnimatedVisualSource, IAnimatedVisualSourceWithUri
```

Comment:
I'm very confused. It looks like `LottieVisualSourceBase` is the only implementer of the interface. Why do we need the interface at all? Can't we just type-check against `LottieVisualSourceBase`?

Reply:
ProgressRing accept a IAnimatedVisualSource as DeterminateSource and IndeterminateSource, it could be a other implementation of LottieVisualSourceBase

Reply:
The @Youssef1313 point is still valid...

Reply:
@Youssef1313 @carldebilly @jeromelaban the implementation is not accessable in Uno.UI so I need the internal interface to access it.

Reply:
Makes sense. Let's merge as is then.